x-pack/legacy/plugins/uptime/public/components/functional/empty_state/empty_state_error.tsx

```diff
@@ -13,25 +13,45 @@ interface EmptyStateErrorProps {
   errors: GraphQLError[];
 }
 
-export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => (
-  <EuiPanel>
-    <EuiEmptyPrompt
-      title={
-        <EuiTitle size="l">
-          <h3>
-            {i18n.translate('xpack.uptime.emptyStateError.title', {
-              defaultMessage: 'Error',
-            })}
-          </h3>
-        </EuiTitle>
-      }
-      body={
-        <Fragment>
-          {errors.map((error: GraphQLError) => (
-            <p key={error.message}>{error.message}</p>
-          ))}
-        </Fragment>
-      }
-    />
-  </EuiPanel>
-);
+export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => {
+  const unAuthorized = errors.find(
```

Comment:
Can we rename this var to `unauthorized`? No need for camelCase here.

Reply:
done